### DISCUSSION TITLE
Can't set audio device on macos 12.6.1

### DISCUSSION THREAD

**Discussion by HotwheelsSisyphus** [espeak-ng/espeak-ng#1568] [Q&A]
Hi, 
I'm noticing that the -d option does nothing on macos 12.6.1 (espeak-ng 1.51.1, installed via homebrew). Whatever I pass as the argument, it just uses the default system device. 
For instance:
`espeak-ng -d "BlackHole\ 2ch" "blah blah blah"`
just plays back over the default system device; 
conversely, if I set Blackhole as the default system device, 
`espeak-ng -d "Built-in\ Output" "blah blah blah"` gets played back over Blackhole, even though it shouldn't.

Any obvious thing that I'm doing wrong, or should I post this as an Issue? (I'm sort of new to all this githubbing, so I don't want to spam the issues board unless I'm sure it's not just me doing something stupid.)

Thanks!

**Comment by jaacoppi** [comment]
Those are not sound devices, just human readable names of sound devices. 

On other unix systems audio device refers to a device file, something like /dev/snd/<id of device>. Apparently MacOS does it differently. See: https://stackoverflow.com/questions/14867033/which-dev-devices-are-the-microphone-and-speaker-in-mac-os-x
